Source/WebCore/dom/ViewTransition.cpp

```diff
@@ -212,6 +212,11 @@ void ViewTransition::skipViewTransition(ExceptionOr<JSC::JSValue>&& reason)
     });
 }
 
+bool ViewTransition::virtualHasPendingActivity() const
```

Comment:
While this may address the use-after-free, it's not the root cause of the issue.

Reply:
I think often, the fix to this sort of crash is:
```
        if (m_isCrossDocument && !isContextStopped())
            m_updateCallbackDone.second->resolve();
```

i.e. not resolving / rejecting promises once `isContextStopped()`

Reply:
As noted, I've changed the approach.